src/main/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorService.scala

```diff
@@ -195,13 +195,6 @@ class PolicyEvaluatorService(
     } yield {
       policies.map(_.groupName)
     }.toSet
-
-  @deprecated("listing policies for resource type removed", since = "ResourceRoutes v2")
```

Comment:
This looked unused when I did a usage search. I happened across it while doing other stuff. Since it was listed as unused and was deprecated, I figured we could just delete it.